docs/workflow.rst

```diff
@@ -15,19 +15,15 @@ To work on a repository project, ``fork`` it first to your git account.
 Your working repositories MUST be cloned from your git account and stored under
```

Comment:
To work on a repository project, ``fork`` it first to your git account. => ``fork`` it first to your personal account.